### PR TITLE
Added a scientific calculator

### PR DESCRIPTION
**Pull request by Girlwithasmile**
### 🛠️ Fixes Issue

#76 

### ✔️ Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
Scientific calculator
Scientific calc preview

### CODE REVIEW

File: Scientific Calculator/manifest.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Scientific Calculator",
+    "version": "0.0.1",
+    "manifest_version": 2,
+    "browser_action": {
+        "default_popup": "index.html",
+        "default_icon":
```
Comment (ridsuteri) [inline Scientific Calculator/manifest.json:7]: add a icon here

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@Girlwithasmile there is some issue within manifest file, please resolve that

**Comment by ridsuteri** [comment]
updates @Girlwithasmile ?

**Comment by kalashvasaniya** [comment]
> updates @Girlwithasmile ?

@ridsuteri if @Girlwithasmile not working on this issue assign this to me :)

**Comment by ridsuteri** [comment]
> > updates @Girlwithasmile ?
> 
> @ridsuteri if @Girlwithasmile not working on this issue assign this to me :)

The existing calculator performs scientific operations too, hence there is no need of this issue now :(

**Comment by kalashvasaniya** [comment]
> > > updates @Girlwithasmile ?
> > 
> > 
> > @ridsuteri if @Girlwithasmile not working on this issue assign this to me :)
> 
> The existing calculator performs scientific operations too, hence there is no need of this issue now :(

okk :)

**Pull request closed without merging**